docs/sources/features/datasources/stackdriver.md

```diff
@@ -131,6 +131,12 @@ It is also possible to choose fixed time intervals to group by, like `1h` or `1d
 
 Group by resource or metric labels to reduce the number of time series and to aggregate the results by a group by. E.g. Group by instance_name to see an aggregated metric for a Compute instance.
 
+#### Metadata labels
```

Comment:
This section is much easier to read now!

Reply:
Nice! Do you want to approve so that I can merge?